packages/ozone-material/ozone-video-player/src/ozone-video-player.ts

```diff
@@ -312,4 +321,18 @@ export class OzoneVideoPlayer extends Polymer.Element {
 		}
 		return null
 	}
+
+	_refreshContent() {
+		if (this.video) {
+			this.loadOzoneVideo(this.video)
+		} else if (this.videoUrl) {
+			this.loadVideoUrl(this.videoUrl)
+		}
+	}
+
+	_playerDimensionsChanged(playerDimensions: IPlayerDimension) {
```

Comment:
you better use player.resize clappr API https://clappr.github.io/classes/Player.html#method_resize.
Because the _refreshContent stop the video